ovos_plugin_common_play/ocp/__init__.py

```diff
@@ -42,11 +41,14 @@ class OCP(OVOSAbstractApplication):
     }
 
     def __init__(self, bus=None, lang=None, settings=None):
-        settings = settings or OCPSettings()
+        # settings = settings or OCPSettings()
         res_dir = join(dirname(__file__), "res")
         gui = OCPMediaPlayerGUI()
         super().__init__(skill_id="ovos.common_play", resources_dir=res_dir,
-                         bus=bus, lang=lang, settings=settings, gui=gui)
+                         bus=bus, lang=lang, gui=gui)
+        if settings:
+            LOG.debug(f"Updating settings from value passed at init")
+            self.settings = settings
```

Comment:
dont replace the internal jsondatabase object, simply merge the dicts and let OVOSAbstractApp handle the saving/reloading and so on. skipping this messes up the filewatcher integration and so on

`self.settings.update(settings)`, same is needed in OCPMediaPlayer class

Reply:
oh you are right, i forgot about that setter, but we dont want the call to .clear() so should still call merge only

Reply:
Ah, missed that.